src/main/java/net/sargue/time/jsptags/FormatSupport.java

```diff
@@ -13,149 +13,160 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
- */ 
+ */
 package net.sargue.time.jsptags;
 
-import javax.servlet.jsp.JspException;
-import javax.servlet.jsp.JspTagException;
-import javax.servlet.jsp.PageContext;
-import javax.servlet.jsp.tagext.TagSupport;
 import java.io.IOException;
 import java.text.DateFormat;
-import java.time.*;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.OffsetDateTime;
+import java.time.OffsetTime;
+import java.time.ZoneId;
 import java.time.format.DateTimeFormatter;
 import java.time.temporal.TemporalAccessor;
 import java.util.Locale;
 
+import jakarta.servlet.jsp.JspException;
+import jakarta.servlet.jsp.JspTagException;
+import jakarta.servlet.jsp.PageContext;
+import jakarta.servlet.jsp.tagext.TagSupport;
+
 /**
- * Support for tag handlers for &lt;formatDate&gt;, the date and time
- * formatting tag in JSTL 1.0.
+ * Support for tag handlers for &lt;formatDate&gt;, the date and time formatting
+ * tag in JSTL 1.0.
```

Comment:
Why? This is just a reformatting change. I prefer to avoid non-relevant changes so the history of a file only contains useful information, as long as it's possible.

Reply:
It seemed like a good idea to have clear documentation.